microsoft-365/admin/add-users/let-users-reset-passwords.md

```diff
@@ -65,13 +65,17 @@ These steps turn on self-service password reset for everyone in your business.
 
 ::: moniker-end
 
-2. At the top of the Settings page select **Security & Privacy**.
+2. At the top of the **Org settings** page, select **Security & Privacy** Tab.
   
-3. Select **Self Service Password Reset**.
+3. Select **Self-service Password Reset**.
+
+4. Under **Self-service password reset**, select **Go to the Azure portal to turn on self-service password reset**.
+
+5. In the left navigation pane, select Users, and then on the Users - all users page, select Password reset.
   
-4. On the Properties page, select **All** to enable it for everyone in your business, and then select **Save**.
+6. On the Properties page, select **All** to enable it for everyone in your business, and then select **Save**.
```

Comment:
Replace with:
6. On the **Properties** page, select **All** to enable it for everyone in your business, and then select **Save**.